src/Controls/src/Core/Platform/Windows/Extensions/CollectionViewExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using WSetter = Microsoft.UI.Xaml.Setter;
+using WStyle = Microsoft.UI.Xaml.Style;
+
+namespace Microsoft.Maui.Controls.Platform
+{
+	public static class CollectionViewExtensions
+	{
+		public static WStyle GetItemContainerStyle(this LinearItemsLayout layout)
```

Comment:
Do we want to be able to return something that is not null for a null layout? Does the following code make sense:
```csharp
var test = (LinearItemsLayout) null;
var style = test.GetItemContainerStyle();
```

Style won't be null yet we have a null object, that is confusing.